site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -155,6 +161,7 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
                         <input type="hidden" name="consequenceType" value="networkWide" />
                         <input type="hidden" name="disruptionId" value={props.disruptionId} />
                         <input type="hidden" name="consequenceIndex" value={props.consequenceIndex} />
+                        <input type="hidden" name="template" value={props.template} />
```

Comment:
As discussed - this line has an unsafe assignment of any value error

Reply:
done